.github/workflows/osv-scanner-scan.yml

```diff
@@ -0,0 +1,34 @@
+# This file is adapted from https://github.com/google/osv-scanner
+
+
+name: OSV-Scanner Scan
+
+on:
+  pull_request:
+    branches:
+    - 'main'
+    - 'release-*'
+  push:
+    branches:
+    - 'main'
+    - 'release-*'
+  schedule:
+  - cron: "12 12 * * 1"
+
+# Restrict jobs in this workflow to have no permissions by default; permissions
+# should be granted per job as needed using a dedicated `permissions` block
+permissions: {}
+
+jobs:
+  scan-scheduled:
+    permissions:
+      contents: read # to fetch code (actions/checkout)
+      security-events: write # for uploading SARIF files
+    if: ${{ github.event_name == 'schedule' }}
```

Comment:
We'd probably want to limit it to this repo too, I don't think every fork should be running this?

Reply:
Yep, I agree, I will push a check here for upstream only